src/Stripe.net/Services/SubscriptionSchedules/SubscriptionScheduleInvoiceSettingsOptions.cs

```diff
@@ -0,0 +1,15 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class SubscriptionScheduleInvoiceSettingsOptions : StripeEntity
```

Comment:
Talking with @alexander-stripe There is a world in which all `invoice_settings` are merged (instead of being 2 separate hashes). This is not yet scheduled/planned though, just some thoughts. This means that when it happens it'd be a major/breaking change anyway.

Still flagging that I am not happy with all those classes names that are attached to the first object ever seeing them...

Reply:
I think last time we tried there was a stylecop rule against nested class.

Reply:
Ah right, thanks. I'm not against changing that rule, but we'd have to be sure it's the right move. stripe-java does use inner classes, so it's worth experimenting with them at least.